example.env

```diff
@@ -6,29 +6,19 @@
 #   2) Rename it to ".env" (which will always be ignored by git)
 #   3) Uncomment the various ENV vars below and provide secrets for your test accounts
 #
-# See the "nox -s "fides_env(test)"" command or scripts/setup/load_examples.py for
-# details on how these are used.
+# See the "nox -s "fides_env(test)"" command or browse the src/fides/data/sample_project/
+# folder (especially sample_connections/sample_connections.yml) for how these are used.
 #
-# Storage options
```

Comment:
nb. this change will remove the ability to programmatically configure storage & messaging providers via ENV vars in the test env, which is why I removed those example variables from this file. These features were (effectively) easter eggs in the test env that I think only Roger and I knew about, so I'm pretty sure it'll be OK to do. I'll double-check with him!

Reply:
Found a solution for this that I don't mind - I'm going to remove all these scripts from this repo (where they _imply_ some degree of maintenance, but we're not maintaining them well...) and re-implement a subset of this in our separate test tooling.